doc/user_guide/transformations.rst

```diff
@@ -396,6 +396,46 @@ Let's plot the results side by side:
     )
     plt.show()
 
+
+Total gradient amplitude (aka Analytic Signal)
+----------------------------------------------
+
+We can also calculate the total gradient amplitude of any magnetic anomaly grid.
+This transformation consists in obtaining the amplitude of the gradient of the
+magnetic field in all the three spatial directions by applying  
+
+.. math::
+    |A(x, y)| = \sqrt{\left( \frac{\partial M}{\partial x} \right)^2 + \left( \frac{\partial M}{\partial y} \right)^2 + \left( \frac{\partial M}{\partial z} \right)^2}
+
+
+We can apply it through the :func:`harmonica.total_gradient_amplitude` function.
+
+.. jupyter-execute::
+
+    tga_grid = hm.total_gradient_amplitude(
+        magnetic_grid_padded
+    )
+
+    # Unpad the total gradient amplitude grid
+    tga_grid = xrft.unpad(tga_grid, pad_width)
+    tga_grid
+
+And plot it:
+
+.. jupyter-execute::
+
+    import verde as vd
+
+    maxabs = vd.maxabs(tga_grid)
+    kwargs = dict(cmap="seismic", vmin=0, vmax=maxabs, add_colorbar=False)
+
+    tmp = tga_grid.plot(cmap="seismic", center=0, add_colorbar=False)
```

Comment:
Since the tga is always positive, I think we can go with viridis here and avoid centering the colorscale around zero:
```suggestion
    tmp = tga_grid.plot(cmap="viridis", add_colorbar=False)
```